.github/workflows/static-binary.yaml

```diff
@@ -28,13 +29,16 @@ jobs:
     steps:
     - name: Checkout
       if: github.event_name == 'repository_dispatch'
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
       with:
+        fetch-depth: 0
```

Comment:
Isn't `fetch-depth: 0` going to break `git describe`?

Reply:
Actually it fixes it. Relevant excerpt from the docs at [actions/checkout](https://github.com/actions/checkout):

```
# Number of commits to fetch. 0 indicates all history for all branches and tags.
# Default: 1
fetch-depth: ''
```